builder/__main__.py

```diff
@@ -179,7 +184,10 @@ def builder(
         run_auditwheel(wheels_dir)
 
         # Check if all wheels are on our min requirements
-        if package_wrong := fix_wheels_unmatch_requirements(wheels_dir):
+        if (
+            package_wrong := fix_wheels_unmatch_requirements(wheels_dir)
+            and exit_code == 0
+        ):
```

Comment:
not scope of this PR, but it prevent to rebuild things after a timeout and just cleanup what we not want